between random: 222633645
between random: 222633637
between random: 222633645
between random: 5061
"    require conn_total_data(the_cid) > 0;
"
quic_server_test_accept_maxdata.ivy: line 316: error: assumption failed
output directory: temp/11725
implementation directory: /quic/aioquic
implementation command: python3 examples/http3_server.py --certificate /quic/aioquic/tests/ssl_cert.pem --private-key /quic/aioquic/tests/ssl_key.pem  -v --host 127.0.0.1 --port 4443 -l /results/temp/aioquic_key.log"
../quic_server_test_accept_maxdata (0) ...
implementation command: ['python3', 'examples/http3_server.py', '--certificate', '/quic/aioquic/tests/ssl_cert.pem', '--private-key', '/quic/aioquic/tests/ssl_key.pem', '-v', '--host', '127.0.0.1', '--port', '4443', '-l', '/results/temp/aioquic_key.log"']
quic_process pid: 16177
timeout 100 ./build/quic_server_test_accept_maxdata seed=633 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
